README.md

```diff
@@ -99,6 +99,18 @@ How to use Universum [↑](#structure-of-this-tutorial)
 Okay, enough philosophy. If you want to just start using `universum` and
 explore it with the help of compiler, set everything up according to the instructions below.
 
+There are two primary ways.
+
+### Implicit path
+
+Add a dependency on `universum-prelude` that provides `Prelude` with all the `universum` functionality.
+
+Also, replace `base` dependency with `base-noprelude` to avoid conflicts.
```

Comment:
IMO, it's completely unfair not to mention the `mixin` approach. Yes, it breaks some tools and for many people it's a critical downside, but apart from that it sounds like the cleanest solution which one should prefer if they don't care about broken tools.

Reply:
Makes sense, I completely forgot that this is also an option and worth enlisting here.